.github/workflows/testing.yml

```
@@ -8,34 +8,36 @@ on:
   workflow_dispatch:
 
 jobs:
-  run-tests-ce:
-    # We want to run on external PRs, but not on our own internal
-    # PRs as they'll be run by the push to the branch.
-    #
-    # The main trick is described here:
-    # https://github.com/Dart-Code/Dart-Code/pull/2375
-    #
-    # Also we want to run it always for manually triggered workflows.
+  run-tests-on-ubuntu-22-04:
     if: (github.event_name == 'push') ||
       (github.event_name == 'pull_request' &&
         github.event.pull_request.head.repo.full_name != github.repository) ||
       (github.event_name == 'workflow_dispatch')
-
-    # We could replace it with ubuntu-latest after fixing the bug:
-    # https://github.com/tarantool/setup-tarantool/issues/37
-    runs-on: ubuntu-22.04
-
     strategy:
       fail-fast: false
       matrix:
-        golang:
-          - '1.20'
-          - 'stable'
-        tarantool:
-          - '1.10'
-          - '2.8'
-          - '2.10'
-          - 'master'
+        golang: ['1.20', 'stable']
+        tarantool: ['1.10']
```

The tests on the Linux should include a bit more versions:
```suggestion
        tarantool: ['1.10', '2.10', '3.4', 'master']
```